core/tests/puppeteer-acceptance-tests/specs/logged-out-user/visit-classroom-index-page.spec.ts

```diff
@@ -20,19 +20,36 @@
 import {UserFactory} from '../../utilities/common/user-factory';
 import testConstants from '../../utilities/common/test-constants';
 import {LoggedOutUser} from '../../utilities/user/logged-out-user';
+import {CurriculumAdmin} from '../../utilities/user/curriculum-admin';
 import {ConsoleReporter} from '../../utilities/common/console-reporter';
+import {ReleaseCoordinator} from '../../utilities/user/release-coordinator';
 
 const DEFAULT_SPEC_TIMEOUT_MSECS = testConstants.DEFAULT_SPEC_TIMEOUT_MSECS;
+const ROLES = testConstants.Roles;
 
 ConsoleReporter.setConsoleErrorsToIgnore([
   /http:\/\/localhost:8181\/access_validation_handler\/can_access_classrooms_page Failed to load resource: the server responded with a status of 404 \(Not Found\)/,
 ]);
 
 describe('Logged-out User', function () {
   let loggedOutUser: LoggedOutUser;
+  let curriculumAdmin: CurriculumAdmin;
+  let releaseCoordinator: ReleaseCoordinator;
 
   beforeAll(async function () {
     loggedOutUser = await UserFactory.createLoggedOutUser();
+    curriculumAdmin = await UserFactory.createNewUser(
+      'curriculumAdm',
+      'curriculum_admin@example.com',
+      [ROLES.CURRICULUM_ADMIN]
+    );
+    releaseCoordinator = await UserFactory.createNewUser(
+      'releaseCoordinator',
+      'release_coordinator@example.com',
+      [ROLES.RELEASE_COORDINATOR]
+    );
+
+    releaseCoordinator.enableFeatureFlag('enable_multiple_classrooms');
```

Comment:
Please await here, else this will too flaky!

Reply:
Done.